packages/mobile/src/account/FiatExchange.tsx

```diff
@@ -24,6 +24,10 @@ function FiatExchange() {
     navigation.navigate(Screens.FiatExchangeAmount, { isAddFunds: true })
   }
 
+  function goToExternalExchanges() {
+    navigation.navigate(Screens.ExternalExchanges)
```

Comment:
Should we use our custom `navigate` function from NavigationService instead?

Reply:
We don't consistently use either throughout the codebase, but I agree that it would make sense to settle on one throughout